Marlin/src/pins/stm32f1/pins_BTT_SKR_MINI_E3_common.h

```diff
@@ -285,24 +278,21 @@
      *   EXP1-8 ----------- EXP2-6   EN2
      *   EXP1-7 ----------- EXP1-5   RED
      *   EXP1-6 ----------- EXP2-8   EN1
-     *   EXP1-5 ----------- EXP1-6   LCD_RST
-     *   EXP1-4 ----------- n/c
+     *   EXP1-5 ----------- n/c
+     *   EXP1-4 ----------- EXP2-3   RESET
      *   EXP1-3 ----------- EXP1-8   LCD_CS
      *   EXP1-2 ----------- EXP1-9   ENC
      *   EXP1-1 ----------- EXP1-7   LCD_A0
      *
-     *    TFT-2 ----------- EXP2-9   SCK
-     *    TFT-3 ----------- EXP2-5   MOSI
+     *    TFT-2 ----------- EXP2-5   SCK
+     *    TFT-3 ----------- EXP2-9   MOSI
      *
      * for backlight configuration see steps 2 (V2.1) and 3 in https://wiki.fysetc.com/Mini12864_Panel/
      */
 
-    #define LCD_PINS_RS              EXP1_03_PIN    // CS
-    #define LCD_PINS_ENABLE                 PA3     // MOSI
```

Comment:
Since these pins are apparently meaningless, we can just remove them.

Reply:
ok